core/domain/acl_decorators.py

```diff
@@ -34,18 +34,42 @@
 
 
 def open_access(handler):
-    """Decorator to give access to everyone."""
+    """Decorator to give access to everyone.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that can
+            also give access to everyone.
+    """
 
     def test_can_access(self, *args, **kwargs):
-        """Gives access to everyone."""
+        """Gives access to everyone.
+
+        Args:
+            *args: *. Arguments.
+            **kwargs: *. Keyword arguments.
```

Comment:
Please remove `*args` and `**kwargs` as we don't cover them in the docstrings. They just signify that function can take any number of arguments and don't provide any necessary info. So, please remove `Args:` section completely here as it isn't required.

Reply:
Sorry @dchen97, I just found that they have been added at other places at well. So, it's totally fine to maintain the consistency in the file and no need to remove.